usage/install-on-desktops.md

```diff
@@ -82,8 +88,28 @@ sudo apt update
 sudo apt install waydroid -y
 ```
 
-Then start Waydroid from the applications menu.
+Then start Waydroid from the applications menu, and press OK for automatic downloads.
+
+      Once the downloads is done:
+        1) restart the system;
+        2) start Waydroid from applications menu.
```

Comment:
Ok. You are the boss and you wrote:

"_No. Waydroid will start after finishing download without **any** of this_".

But, the second option is necessary to start WayDroid.


Just some words.

When I did these and others addings, I wanted just to contribute with WayDroid project, that I admire and also I´m grateful for. Thus, following the open source philosophy, that I appreciate a lot, I tried to help, based on tests and taking into consideration to facilitate life of beginner users, avoiding for them complicated actions of restarting container in some cases for example.

Sorry if I didn't please you.

My action is just to contribute and help if it is a request good for all.

If my contribution isn't accepted, no problem. My gratitude remains. I don't want to get in the way, just contribute.

Reply:
Waydroid will start automatically after clicking the "Done" button once the download finishes. There's no need to launch it again from the application menu.

It doesn't show anything until it has finished booting so that might be why you thought more actions were needed.

Reply:
Thank you Astone